src/components/AddBookForm.js

```diff
@@ -37,23 +34,22 @@ const AddBookForm = () => {
     }));
   };
 
-  const { title, author, category } = book;
+  const { title, category } = book;
   const dispatch = useDispatch();
 
   const submitToStore = () => {
     const newBook = {
       ...book,
-      id: uuidv4(),
+      item_id: uuidv4(),
     };
-    dispatch(addBook(newBook));
+    dispatch(sendToServer(newBook));
   };
 
   return (
     <div>
       <h1>Add New Book</h1>
       <form>
-        <input name="title" type="text" placeholder="Book title" value={title} onChange={hangleChange} className="book-title" />
-        <input name="author" type="text" placeholder="Author" value={author} onChange={hangleChange} className="book-author" />
+        <input name="title" required type="text" placeholder="Book title" value={title} onChange={hangleChange} className="book-title" />
         <select name="category" value={category} onChange={hangleChange} id="books">
           {Options()}
         </select>
```

Comment:
- [ ] Great work to implement the `connect to API milestone`, however, let us fix some issues before merging our branch.

- In our form, we are missing the author input.
- if a user adds or sends information, the input has to be empty so that a user should add other information.
- Let us validate the input so that the user couldn't add empty information.
- Let us use es6 syntax as well as arrow function for action creators and define reducers.


![user-15](https://user-images.githubusercontent.com/55635977/140266557-fb2828b9-60cd-4df0-bbe0-6e1da4ec0afd.png)

Reply:
@Emmyn5600 

Thanks for the feedback,
I have fixed the issue with validating input and sending information when the title is empty.

However, on the request to add an author input, the bookstore API doesn't store or return the field, Kindly review the [documentation](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) to verify.

Lastly the request to use es6 syntax as well as arrow function for action creators and define reducers, I do not think it exactly follows the [duck pattern](https://github.com/erikras/ducks-modular-redux) we were instructed to use as part of the requirement.

Please share your thoughts on the concerns I have raised.

Thanks.

Reply:
@Emmyn5600 

Thanks for your understanding.

Reply:
Yeah sure, you add it later in the next milestone.